debian/changelog

```diff
@@ -0,0 +1,5 @@
+percona-pg-telemetry (1.0.0-1) unstable; urgency=medium
```

Comment:
@artemgavrilov Could you please push the release tag , `1.0.1 ` similar to  https://github.com/percona/telemetry-agent/releases/tag/v1.0.1, so that we can be in sync with telemetry-agent.

Reply:
1.0.0 is the version and 1 is the release version for debian/rpm packages. For e.g. `percona-pg-telemetry17-1.0.0-1.el8.aarch64.rpm`.